Front-end/js/main.js

```diff
@@ -35,11 +35,11 @@ let teamNames = ["Red", "Blue", "Green", "Yellow", "Purple", "Cyan", "Orange", "
 
 document.addEventListener('keyup', (event) =>
 {
-    if(event.code == 113) //F2 is our debug key.
+    if(event.key == "F2") //F2 is our debug key.
```

Comment:
Maybe we can remove the comments here as they arent needed with the string syntax